.github/actions/do_build_ock/do_build_m1/action.yml

```diff
@@ -0,0 +1,22 @@
+name: build-ock-m1
+description: Action to build the oneapi-construction-kit for default riscv m1 target
+
+inputs:
```

Comment:
So does this action hide the inputs of `do_build_ock`, meaning we can't have a `do_build_m1` that also sets `inputs.offline_kernel_tests`?

Reply:
right now no, but we can add them as needed.